test/e2e/atlas/search_nodes_test.go

```diff
@@ -0,0 +1,200 @@
+
+		require.NoError(t, err, resp)
+		var searchNode atlasv2.ApiSearchDeploymentResponse
+		require.NoError(t, json.Unmarshal(resp, &searchNode))
+
+		assert.Equal(t, *searchNode.GroupId, g.projectID)
+		assert.Equal(t, searchNode.Specs, pointer.Get([]atlasv2.ApiSearchDeploymentSpec{
+			{
+				InstanceSize: "S30_LOWCPU_NVME",
+				NodeCount:    2,
+			},
+		}))
+	})
+
+	t.Run("List + verify created", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			clustersEntity,
+			searchEntity,
+			nodesEntity,
+			"list",
+			"--clusterName", g.clusterName,
+			"--projectId", g.projectID,
+			"-o=json",
+		)
+
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		resp = bytes.TrimLeft(resp, ".")
+
+		require.NoError(t, err, resp)
+		var searchNode atlasv2.ApiSearchDeploymentResponse
+		require.NoError(t, json.Unmarshal(resp, &searchNode))
+
+		assert.Equal(t, *searchNode.GroupId, g.projectID)
+		assert.Equal(t, searchNode.Specs, pointer.Get([]atlasv2.ApiSearchDeploymentSpec{
+			{
+				InstanceSize: "S30_LOWCPU_NVME",
+				NodeCount:    2,
+			},
+		}))
+	})
+
+	t.Run("Update search node", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			clustersEntity,
+			searchEntity,
+			nodesEntity,
+			"update",
+			"--clusterName", g.clusterName,
+			"--projectId", g.projectID,
+			"--file", "search_nodes_spec_update.json",
+			"-w",
+			"-o=json",
+		)
+
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		resp = bytes.TrimLeft(resp, ".")
+
+		require.NoError(t, err, resp)
+		var searchNode atlasv2.ApiSearchDeploymentResponse
+		require.NoError(t, json.Unmarshal(resp, &searchNode))
+
+		assert.Equal(t, *searchNode.GroupId, g.projectID)
+		assert.Equal(t, searchNode.Specs, pointer.Get([]atlasv2.ApiSearchDeploymentSpec{
+			{
+				InstanceSize: "S20_HIGHCPU_NVME",
+				NodeCount:    3,
+			},
+		}))
+	})
+
+	t.Run("List + verify updated", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			clustersEntity,
+			searchEntity,
+			nodesEntity,
+			"list",
+			"--clusterName", g.clusterName,
+			"--projectId", g.projectID,
+			"-o=json",
+		)
+
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		resp = bytes.TrimLeft(resp, ".")
+
+		require.NoError(t, err, resp)
+		var searchNode atlasv2.ApiSearchDeploymentResponse
+		require.NoError(t, json.Unmarshal(resp, &searchNode))
+
+		assert.Equal(t, *searchNode.GroupId, g.projectID)
+		assert.Equal(t, searchNode.Specs, pointer.Get([]atlasv2.ApiSearchDeploymentSpec{
+			{
+				InstanceSize: "S20_HIGHCPU_NVME",
+				NodeCount:    3,
+			},
+		}))
+	})
+
+	t.Run("Delete search nodes", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			clustersEntity,
+			searchEntity,
+			nodesEntity,
+			"delete",
+			"--clusterName", g.clusterName,
+			"--projectId", g.projectID,
+			"--force",
+			"-w",
+			"-o=json",
+		)
+
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		resp_str := strings.TrimLeft(string(resp), ".")
+
+		require.NoError(t, err, resp_str)
```

Comment:
```suggestion
		require.NoError(t, err, string(resp))
```

do we need the `trimLeft`?

Reply:
Same as https://github.com/mongodb/mongodb-atlas-cli/pull/2654#discussion_r1494909122

>  When we're not in a terminal (which is the case when running e2e tests), we print dots to show progress.
> Caused by:
https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/cli/watch_opts.go#L56
> This makes your output look like this: `.....{ /* json */ }`, that's why I'm trimming the dots on the left side